hollow-explorer-ui/src/main/java/com/netflix/hollow/explorer/ui/HollowExplorerUI.java

```diff
@@ -85,8 +88,14 @@ public boolean handle(String target, HttpServletRequest req, HttpServletResponse
         } else if("query".equals(pageName)) {
             queryPage.render(req, resp, session);
             return true;
+        } else if("go".equals(pageName)) {
+            // The target has / prefixed in namespaceURL, this is to escape it.
+            if(target.startsWith("/")) {
+                target = target.substring(1);
+            }
+            URL namespaceURL = new URL("http://"+target);
```

Comment:
I don't thing we should say anything about namespace here, right? Since hollow has no concept of namespace. This is really a generic url for the UI.
Also, not entirely sure what we're doing by stripping out the slash, are we trying to disallow absolute urls for some reason?

Reply:
You're right. I was just trying to construct an absolute URL from the one being passed from the page. The target I receive from the explorer page is "/go/cinderdashboard...". It was a hacky way to get the link working. The namespaceURL part isn't final yet. Still figuring out a clean way to get it running from explorer link.